lib/edmunds/vehicle/specification/color.rb

```diff
@@ -10,11 +10,16 @@ module Specification
       module Color
         
         class ColorsByStyle
-          attr_reader :colors, :count
+          attr_reader :exterior, :interior #, :colors, :count
 
           def initialize(attributes)
-            @colors = attributes['colors'].map { |json| ColorsDetails.new(json) } if attributes.key?('colors')
-            @count = attributes['colorsCount']
+            # @colors = attributes['colors'].map { |json| ColorsDetails.new(json) } if attributes.key?('colors')
```

Comment:
@fabrouy Sorry for the dumb question but is this the result of a more recent API change?